reference/7/Microsoft.PowerShell.Utility/Get-Error.md

```diff
@@ -0,0 +1,144 @@
+---
+external help file: Microsoft.PowerShell.Commands.Utility.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Utility
+ms.date: 11/12/2019
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/get-error?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-Error
+---
+
+# Get-Error
+
+## SYNOPSIS
+Gets and displays the most recent error messages from the current session.
+
+## SYNTAX
+
+```
+Get-Error [[-InputObject] <psobject>] [-Newest <Int>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-Error` cmdlet gets a **PSExtendedError** object that represents the current error details
+from the last error that occurred in the session.
+
+You can use 'Get-Error' to display a specified number of errors that have occurred in the current
+session using the **Newest** parameter.
+
+The 'Get-Error' cmdlet also receives error objects from a collection, such as **$Error**, to display
+multiple errors from the current session.
+
+## EXAMPLES
+
+### Example 1: Get the most recent error details
+
+In this example, `Get-Error` displays the details of the most recent error that occurred in the
+current session.
+
+```powershell
+Get-Childitem -path c:\NoRealDirectory
```

Comment:
What about making this non-Windows specific? 

```
Get-Childitem -path /NoRealDirectory
```

This works on all three OS targets.